docs/index.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml">
     <head>
-        <meta http-equiv="Refresh" content="0; url='html/index.html'" />
+        <meta http-equiv="Refresh" content="0; url='../home/html/index.html'" />
```

Comment:
This seems wrong.  You want to redirect from docs/index.html to home/html/index.html?  I would think after the last Pull Request, **this** file was generated.  Further, I don't see a home anywhere in your branch.

Reply:
Fixed.